### ISSUE TITLE
setting discord config and in metadata causes author and pack to be ignored by sticker api

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2642]

**Comment by t0g3pii** [comment]
[Sub]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.34.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.34.0...4.34.1) (2022-04-10)

- Release 4.34.1 [`77c3ea0`](https://github.com/open-wa/wa-automate-nodejs/commit/77c3ea0084d5a5e482a97349b0838bea04a1d3a3)
- 🩹 fix sticker author/pack +discord issue #2642 [`69bb79c`](https://github.com/open-wa/wa-automate-nodejs/commit/69bb79c956a3f968489543503fad6e551552aaa8)
- updated types-only package [`ff43624`](https://github.com/open-wa/wa-automate-nodejs/commit/ff436242de66cc659cf9ae32e4dddd969723e65c)

**Comment by t0g3pii** [comment]
+1 works like a charm
